spec/models/repository_spec.rb

```diff
@@ -25,33 +26,46 @@
         expect(environment.name).to eq 'staging'
       end
     end
+  end
+
+  describe '#default_environment' do
+
+    before do
+      @repo = Repository.with_name('acme-inc/api')
+      @repo.configure! <<-YAML
+environments:
```

Comment:
IMO, it would probably be better to make setting the default environment more explicit.

Either this:

```yaml
environments:
  production:
    default: true
    aliases: [prod]
  stage:
    aliases: [staging]
```

Or this:

```yaml
default_environment: production
environments:
  production:
    aliases: [prod]
  stage:
    aliases: [staging]
```

I like the latter, because it's more obvious that you can only have 1 default environment (which also makes implementing it easier).